network-api/networkapi/wagtailpages/pagemodels/buyersguide/homepage.py

```diff
@@ -376,6 +377,7 @@ def get_context(self, request, *args, **kwargs):
 
         context['categories'] = get_categories_for_locale(language_code)
         context['current_category'] = None
+        context['pagetype'] = 'home'
```

Comment:
@fessehaye Do we still need this information after the latest changes in the template?